jobs/config.json

```diff
@@ -9455,7 +9456,7 @@
     ],
     "scenario": "kubernetes_verify",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-testing"
```

Comment:
I still very much dislike this job *very* much but it does seem sig-testing owns it :joy: :shipit:
/shrug

Reply:
in fact every job using the `kubernetes_verify` is a pile of dirty hacks owned by sig-testing 🤷‍♂️